compiler/z/codegen/OpMemToMem.cpp

```diff
@@ -125,7 +125,7 @@ MemToMemVarLenMacroOp::generateLoop()
          _raReg = _cg->allocateRegister();
 
       //use literal for aot to make it easier for relocation
-      if (comp->compileRelocatableCode())
+      if (comp->fej9()->helpersNeedRelocation())
```

Comment:
We can do something similar to the `cg->needRelocationsForStatics()` query. It can return `false` at the OMR level, and do something like the following in OpenJ9:
https://github.com/eclipse/openj9/blob/94c6cde79bfa25d8c306acc4bc7bef080fc717ee/runtime/compiler/codegen/J9CodeGenerator.cpp#L4658-L4662